cmd/grapi-gen-service/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/svcgen"
+	"github.com/izumin5210/grapi/pkg/svcgen/template"
+)
+
+func main() {
+	gencmd.Main(
+		"service",
+		gencmd.WithGenerateCommand(NewGenerateCommand(svcgen.NewApp)),
+	)
+}
+
+type CreateAppFunc func(*gencmd.Ctx, *gencmd.Command) (*svcgen.App, error)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CreateAppFunc should have comment or be unexported